index.html

```diff
@@ -1,87 +1,108 @@
+        integrity="sha384-YvpcrYf0tY3lHB60NNkmXc5s9fDVZLESaAA55NDzOxhy9GkcIdslK1eN7N6jIeHz"
+        crossorigin="anonymous"></script>
+    <link href="./src/css/index.css" rel="stylesheet" />
+    <link href="./src/css/shared.css" rel="stylesheet" />
+    <style>
+        @import url("https://fonts.googleapis.com/css2?family=Dosis:wght@200..800&display=swap");
+    </style>
+</head>
 
-  <body>
-    <section class="hero" id="home">
-      <div class="title">
-        <img src="resources/images/早安.png" class="circle-image">
-        <h1>RedcXca</h1>
-        <p>Hi! I am a second year software engineering student at the University of Waterloo. I do digital art and play osu!</p>
-        <svg></svg><br>
-        <a class="commission-btn box-bold" href="https://docs.google.com/document/d/1CELGlKdLDq_-Usue3oQfcrawi-vKOXa3SOkzTHnqB5A/edit#heading=h.9lj1hkizyon2" target="_blank">Commission Info</a>
-        <div class="external-links"><br>
-          <ul>
-            <li><a id="pixiv" href="https://www.pixiv.net/en/users/49573484" target="_blank" onmouseover="ExternalLinksEnter('pixiv')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/pixiv.svg"><p>Pixiv</p></a></li>
-            <li><a id="instagram" href="https://www.instagram.com/RedcXca/" target="_blank" onmouseover="ExternalLinksEnter('instagram')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/instagram.svg"><p>Instagram</p></a></li>
-            <li><a id="twitter" href="https://twitter.com/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('twitter')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/twitter.svg"><p>Twitter</p></a></li>
-            <li><a id="osu" href="https://osu.ppy.sh/users/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('osu')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/osu.svg"><p>osu!</p></a></li>
-            <li><a id="github" href="https://github.com/RedcXca" target="_blank" onmouseover="ExternalLinksEnter('github')" onmouseleave="ExternalLinksLeave()"><img src="resources/svg/github.svg"><p>GitHub</p></a></li>
-          </ul>
+<body>
+    <div class="hero"></div>
+    <nav id="navbar">
+        <div class="logo">RedcXca</div>
+        <div class="nav-link-wrapper">
+            <ul class="nav-links">
+                <li><a href="./index">About</a></li>
+                <li><a href="./projects">Projects</a></li>
+                <li><a href="./illustrations">Illustrations</a></li>
+                <li><a href="./osu">osu!</a></li>
+            </ul>
+        </div>
+    </nav>
+    <header>
+        <img src="src/images/早安.png" class="circle-image">
+        <section>
+            <h1 class="title">Max Lu/RedcXca</h1>
+            <div class="external-links">
+                <ul>
+                    <li><a id="osu" href="https://osu.ppy.sh/users/RedcXca" target="_blank"
+                            onmouseover="ExternalLinksEnter('osu')" onmouseleave="ExternalLinksLeave()"><img
```

Comment:
invert the colours of this svg ;w; also maybe move the buttons a bit closer

Reply:
![image](https://github.com/user-attachments/assets/1130619e-48bd-414a-96f2-666d9f70244d)
Edited the SVG to be white.

Reply:
![image](https://github.com/user-attachments/assets/c6df1578-19a6-4a03-bae1-7d1e5db3e1f2)
Made the buttons closer and align